### PR TITLE
Add script to import SCMD data

### LINKED ISSUE #2831: Upload Secondary Care Medicines Data dataset to BQ
Monthly data has been released at https://opendata.nhsbsa.net/dataset/secondary-care-medicines-data, and we want this in BQ.

Suggest uploading to a table in a new dataset, calling both the dataset and the table `scmd`.  The table should have the following schema:

* `year_month`: `date` (we'll need to convert eg `2019-01` to `2019-01-01`)
* `ods_code`: `string`
* `vmp_snomed_code`: `string`
* `vmp_product_name`: `string`
* `unit_of_measure_identifier`: `string`
* `unit_of_measure_name`: `string`
* `total_quanity_in_vmp_unit: `float`

The data should be uploaded by a management command in the [`pipeline`](https://github.com/ebmdatalab/openprescribing/tree/master/openprescribing/pipeline/management/commands) app.  There is a wrapper around the Google SDK client at [`gcutils.bigquery.client`](https://github.com/ebmdatalab/openprescribing/blob/master/openprescribing/gcutils/bigquery.py) that can do most of the heavy lifting.

The command should:

* scrape the months that have been published
* check which months have already been uploaded (using [`gcutils.bigquery.client.query`](https://github.com/ebmdatalab/openprescribing/blob/master/openprescribing/gcutils/bigquery.py#L212-L221)
* append to the table data for any months that have not yet been uploaded (using  [`gcutils.bigquery.client.insert_rows_from_csv`](https://github.com/ebmdatalab/openprescribing/blob/master/openprescribing/gcutils/bigquery.py#L325-L339), with `write_disposition=WRITE_APPEND`)

If time is short, we can start with a script that just uploads all the existing data, and we can work on keeping it updated later.

### PR DESCRIPTION
**Pull request by ghickman**
This adds a script to import the monthly Secondary Care Medicines Data datasets from NHSBSA into BigQuery.

I've run the script locally and 2019-01 -> 2020-07 are now in BigQuery.

Fixes #2831

### CODE REVIEW

File: openprescribing/pipeline/management/commands/import_scmd.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+        doc = BeautifulSoup(r.text, "html.parser")
+
+        for a in doc.find_all("a", href=True):
+            if a["href"].endswith(".csv"):
+                yield a["href"]
+
+    def iter_months(self, urls):
+        """
+        Extract a two-digit month from each URL given.
```
Comment (inglesp) [inline openprescribing/pipeline/management/commands/import_scmd.py:112]: It's not a two-digit month?

File: openprescribing/pipeline/management/commands/import_scmd.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+                # insert into BigQuery
+                table.insert_rows_from_csv(
+                    f.name, SCHEMA, write_disposition="WRITE_APPEND"
+                )
+                print("{} | Ingested into BigQuery".format(month))
+
+    def iter_dataset_urls(self, session):
+        """Extract CSV file URLs from the dataset page."""
+        dataset_page = (
```
Comment (inglesp) [inline openprescribing/pipeline/management/commands/import_scmd.py:96]: This is a URL.

**Review by inglesp — APPROVED** [review]
Good stuff, lots of different moving parts to navigate.  A couple of minor naming nits.

File: openprescribing/pipeline/management/commands/import_scmd.py (hunk 1 above)
Comment (ghickman) [reply openprescribing/pipeline/management/commands/import_scmd.py:112]: Oops, stale comment!

File: openprescribing/pipeline/management/commands/import_scmd.py (hunk 2 above)
Comment (ghickman) [reply openprescribing/pipeline/management/commands/import_scmd.py]: Fixed in e8bf6ad6

File: openprescribing/pipeline/management/commands/import_scmd.py (hunk 1 above)
Comment (ghickman) [reply openprescribing/pipeline/management/commands/import_scmd.py]: Fixed in e8bf6ad6

**Pull request merged**